docker-assets/manageiq.conf

```diff
@@ -0,0 +1,114 @@
+<Proxy balancer://evmcluster_ui/ lbmethod=bybusyness>
+  BalancerMember https://manageiq
```

Comment:
Should these bits be dynamically written from a `MANAGEIQ_SERVICE_NAME` environment variable?

I feel like that might be a bit more work but is probably the "right" way to do it.